stdlib/logging/__init__.pyi

```diff
@@ -31,13 +31,27 @@ class Filterer(object):
     def removeFilter(self, filter: _FilterType) -> None: ...
     def filter(self, record: LogRecord) -> bool: ...
 
+class Manager(object):
+    root: RootLogger
+    disable: int
+    emittedNoHandlerWarning: bool
+    loggerDict: dict[str, Logger]
+    loggerClass: Optional[Logger]
+    logRecordFactory: Optional[LogRecord]
```

Comment:
`loggerClass` should be a class inheriting from `Logger` not literally a `Logger` instance. Also, `logRecordFactory` should be typed similarly to global `getLogRecordFactory` and `setLogRecordFactory` functions:

```suggestion
    loggerClass: Optional[Type[Logger]]
    logRecordFactory: Callable[..., LogRecord]
```

This also applies to `setLoggerClass` and `setLogRecordFactory`.